recipes/libftdi/all/test_package/conanfile.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake_find_package"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package"
```

Reply:
#### Done

`cmake` added to generators list within test_package conanfile.